microdonuts/src/main/java/com/otsample/api/KitchenConsumer.java

```diff
@@ -31,23 +31,9 @@
     public KitchenConsumer()
     {
         // A decorator that overrides the operation name with the URL path
-        SpanDecorator opNameDecorator = new SpanDecorator() {
```

Comment:
Wondering why you have removed the custom `SpanDecoarator` here and in the KitchenContextHandler? In this case, they are useful to show how to override the operation name.